nailgun/entities.py

```diff
@@ -844,9 +844,11 @@ def __init__(self, server_config=None, **kwargs):
             'command': entity_fields.StringField(
                 required=True,
                 str_type='latin1',
+                default='top',
```

Comment:
Actually we should talk more about this. As default image is busybox then the top command will be available. But regarding docker the command should be passed explicitly.